kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/ingress/ClusterIPIngressDefinition.java

```diff
@@ -97,29 +107,158 @@ public Stream<ContainerPort> proxyContainerPorts() {
                     });
             return Stream.concat(bootstrapPort, ingressNodePorts);
         }
+
+        @Override
+        public boolean requiresSharedTLSPort() {
```

Comment:
Would the method be better named `requiresSharedPort` or `requiresUniquePorts` (obviously that inverts the booleans but might read better at call sights)